cmd/thor/pruner/pruner.go

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2019 The VeChainThor developers
```

Comment:
should we put new date ?

Reply:
moved from `cmd/thor/optimizer/optimizer.go`